symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py

```diff
@@ -28,15 +30,9 @@
     validate_output,
 )
 
-effective_mass = Symbol("mass", units.mass)
+effective_mass = clone_symbol(symbols.basic.mass, "effective_mass", display_symbol="m_eff", display_latex=r"m_\text{eff}")
```

Comment:
Remove "effective_mass" text.
Use `\\` in latex and avoid r-strings.

Reply:
Why not r-strings?

Reply:
I don't like em.

Reply:
To clarify, you only mean r-strings in this context or everywhere?